### ISSUE TITLE
Collapsed Sidebar Expands When Interacting With Browser

### ISSUE THREAD

**Issue by jall-in** [RinteRface/bs4Dash#102]
Hey and  thanks a lot for this great package! I stumbled over some weird behaviour which I couldn't see in earlier releases of this package. When you expand a collapsed card item and the (left) sidebar is collpased at the same time, then this sidebar is also being extended. The same strange behaviour can be observed when you resize the browser window which means that if, again, the sidebar is collapsed and you resize your browser window (or enter developers mode), then the sidebar is being extended again. 

Any way to fix this? For a reproducible example run the example application and follow my description above. https://dgranjon.shinyapps.io/bs4DashDemo/_w_c1e4846d/

**Comment by martinv13** [comment]
Thank you also for this package. I have this problem too.

If it can help, I think the problem happened while updating AdminLTE in october 19. It happened somewhere commits 8ef2dd92 and 7077f22b (I looked it up using the `bs4DashGallery()` demo.

Also, the demo used to start with sidebar collapsed before that, and is now expanded on startup.

Sorry I don't understand the internals and interaction with AdminLTE well enough to drill down further the issue, I'll keep trying though as this is quite a big UX issue in my case.
Thanks.

**Comment by jall-in** [comment]
I digged a little bit into the code and could narrow it down to the AdminLTE javascript, which has been replaced by a customized script.  
So what happens is that the collapse / show functions (responsible for the toggeling of the sidebar) are called for example on a window resize event. In the javascript the classes open / collapsed are then removed or appended. However, the class open is deprecated. Would be great to get this fixed! Thanks.
